### PR TITLE
Timeline visual. Fixed #2340 cursors position.

### PR DESCRIPTION
**Pull request by tomsoir**

### CODE REVIEW

File: src/Clients/CustomVisuals/visuals/timeline/timeline.ts [hunk 1]
```diff
@@ -605,6 +610,12 @@ module powerbi.visuals.samples {
 
             timelineData.timelineDatapoints = [];
 
+            var dX2 = timelineFormat.cellWidth * (timelineData.elementsCount + 1);
```
Comment (v-kazaki) [inline src/Clients/CustomVisuals/visuals/timeline/timeline.ts]: Use {} for conditional blocks. Check coding guidelines - https://github.com/Microsoft/PowerBI-visuals/wiki/Coding-Guidelines#do-surround-loop-and-conditional-bodies-with-

File: src/Clients/CustomVisuals/visuals/timeline/timeline.ts [hunk 2]
```diff
@@ -706,58 +709,47 @@ module powerbi.visuals.samples {
...
-                    if (dateStartSelectTime === dateEndTime && timelineData.granularity !== 'day') {
-                        timelineData.cursorDataPoints[0].dx = dX + cellWidth;
-                    }
-                    if (dateEndSelectTime === dateStartTime && timelineData.granularity !== 'day') {
-                        timelineData.cursorDataPoints[1].dx = dX;
-                    }
-                    if (dateEndSelectTime === dateEndTime && timelineData.granularity !== 'day') {
-                        timelineData.cursorDataPoints[1].dx = dX + cellWidth;
+                        }
+                        if (dateStartSelectTime === dateStartTime) timelineData.cursorDataPoints[0].dx = dX;
```
Comment (v-kazaki) [inline src/Clients/CustomVisuals/visuals/timeline/timeline.ts]: Same here.

**Comment by v-kazaki** [comment]
Other changes look good.

**Comment by v-kazaki** [comment]
Please rename PR so it will be clear what was issues were fixed.

**Comment by tomsoir** [comment]
@v-kazaki 
1) Fix coding guidelines comments
2) Squash commits to one

**Comment by tomsoir** [comment]
@v-kazaki 
1) fixed all of your PR comments
2) squash commits to one
Thank you.

**Pull request closed without merging**